src/typography/_colors.scss

```diff
@@ -0,0 +1,16 @@
+:root {
+  --primary-green: #13aa57;
+  --secondary-dark-green: #056b33;
+  --tertiary-light-green: #39b54a;
+  --primary-light-grey: #f5f7f6;
+  --secondary-light-grey: #f5f6f6;
+  --error-ligh-red: #f03127;
+  --first-color-font: #494a49;
+  --second-color-font: #839c94;
+  --third-font-color: #bebebe;
+  --fourth-font-color: #263238;
+  --fifth-font-color: #313131;
+  --sixth-font-color: #2b292d;
+  --seventh-font-color: #878787;
+  --eighth-font-color: #666;
```

Comment:
good job,
but how about using clearer names? instead of --fifth-font-color  or --seventh-font-color...as light gray, dark gray, etc.
also, use 1 style --second-color-font or --third-font-color...

Reply:
done